client/packages/programs/src/VaccineCourseEditModal/VaccineCourseEditModal.tsx

```diff
@@ -259,6 +259,7 @@ const VaccineCourseDoseTable = ({
           // temp - will be overwritten by the backend to assign unique dose number (even if previous doses were deleted)
           label: `${courseName} ${doses.length + 1}`,
           minAgeMonths: (previousDose?.minAgeMonths ?? 0) + 1,
+          maxAgeMonths: previousDose?.minAgeMonths ?? 1,
```

Comment:
Shouldn't this have a +2, or be prev.maxAge +1 or something? Otherwise maxAge is less than min age when you add a row?

Reply:
Yeah, I thought maybe we were going to be getting new defaults from back-end, but never mind, have updated now.